### DISCUSSION TITLE
NBMiner v42.1

### DISCUSSION THREAD

**Discussion by nebutech-admin** [NebuTech/NBMiner#890] [General]
`fix`: `ethash` Linux LHR unlocker can not unlock on certain GPUs.

<hr /><em>This discussion was created from the release <a href='https://github.com/NebuTech/NBMiner/releases/tag/v42.1'>NBMiner v42.1</a>.</em>

**Comment by DDRiDIM** [comment]
Через 60 минут перезапуск майнера на всех фермах. Почему так? в версии 41.5 такого нет.

**Reply by DDRiDIM** [reply]
power limit 100%. clock fix. Spare power supplies. What else does he need?

**Comment by drvrx** [comment]
Very nice! Any chance of getting share diffs in the future?

**Comment by FerrinMass** [comment]
Much more power hungry than 41.5. Power consumption went up by about 2%, hashrate went up by 0.17%. Not happy about the extra power need.

**Reply by DDRiDIM** [reply]
power limit 100%. clock fix. Spare power supplies. What else does he need?

**Comment by Amishtek** [comment]
will this version also work with previous nvidia drivers? I would like to stay on 511 as 512 has serious issues with my system causes BOD

**Comment by aminlv** [comment]
driver install uninstall get error 10

**Comment by jwiselog** [comment]
Any plans on including 3090 ti LHR unlock support soon?
Edit: Lol nvm i just have some weird OC or hardware issue.

**Comment by jekeam** [comment]
ETH - At RTX 3050, the hashrate became 12 Mh (in NBMiner v41.5 it was about 22 Mh), why did this happen? (with 512.95 drivers)

**Reply by jwiselog** [reply]
Im not the dev, but check your nvidia driver version and roll back if necessary. This is a common issue in the past few days with the .77 driver from nVidia

**Comment by Eb1n1zer** [comment]
LHR unlock stopped working on RTX 3080Ti video card with 512.95 drivers

**Comment by hushpuppyminer01** [comment]
Im try on this Nbminer Version 42.1 look good at My Card 3060 v1 ND 512.15 Hashrate 48.8 Max

**Reply by hushpuppyminer01** [reply]
Running 1 day client mining crash again. Test next version 42.2
